tasks/components/cards/PatientCard.tsx

```diff
@@ -33,27 +50,39 @@ export const PatientCard = ({
   overwriteTranslation,
   bedName,
   patientName,
-  unscheduledTasks,
-  inProgressTasks,
-  doneTasks,
+  openTasks = 0,
+  gender = 'diverse',
+  hasWarning = false,
+  age = 32,
```

Comment:
Please do not set random values as default. Add a comment If thats a temporary solution until services is ready.

Reply:
You are right, I will move this PR to draft status for now until the backend has implemented this